runelite-api/src/main/java/net/runelite/api/events/ChatMessage.java

```diff
@@ -28,6 +28,10 @@
 import lombok.Data;
 import net.runelite.api.ChatMessageType;
 
+/**
+ * This event is triggered whenever a chat message appears in the chat box or in private message. This also happens
```

Comment:
I would remove the "This also happens whenever the player sends a message." part, since when you send a message it also appears in the chatbox/PM so the previous statement is enough IMO.